java/25/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM eclipse-temurin:25_36-jdk-noble AS base
+
+LABEL maintainer="opensource@ekino.com"
+LABEL org.opencontainers.image.source="https://github.com/ekino/docker-buildbox/"
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    LANG=C.UTF-8
+
+FROM base AS base-amd64
+ARG AWSCLI_ARCH="linux-x86_64"
+
+FROM base AS base-arm64
+ARG AWSCLI_ARCH="linux-aarch64"
+
+FROM base-${TARGETARCH}
+
+# Updating packages using sources
+RUN cat /etc/apt/sources.list && \
+    apt-get -qq clean -qq && apt-get -qq update && \
+    # Install base
+    apt-get -qq -y install build-essential curl git make openssh-client jq unzip groff less && \
```

Comment:
Since it's a new image, we could try to trim the packages a bit: I think we can get rid of `build-essential`, `make`, even `groff` and `less` if it's not used interactively.

We could also group the `graphviz` and `postgresql-client` installs with this one.